openid-4-verifiable-presentations-1_0.md

```diff
@@ -746,7 +752,15 @@ The following is a non-normative example of the payload of the JWT used in the e
 
 <{{examples/response/jarm_jwt_vc_json_body.json}}
 
-## Error Response
+## Transaction Data {#transaction_data}
+
+Transaction data mechanism enables binding between the user identification/authentication and user’s authorization, for example to complete a payment transaction, or to sign specific document(s) using QES (Qualified Electronic Signatures). This is achieved by signing the transaction data used for user authorization with the user-controlled key used for proof of possession of the Credential being presented as a means for user identification/authentication.
+
+The Wallet that received `transaction_data` parameter in the request, MUST include in the response a `tx_data_hashes` parameter defined below: 
```

Comment:
Why add this to the response and not just to the KB-JWT as defined below? If that is what is meant, this should be clarified here.

Reply:
the parameter is common throughout multiple credential formats and the only difference is where this credential is included. will clarify this

```suggestion
The Wallet that received the `transaction_data` parameter in the request MUST include in the response a `tx_data_hashes` parameter defined below. Where `tx_data_hashes` parameter is included in the response is specific to each credential format and is defined in a respective Credential Format Profile.
```